src/main/java/github/scarsz/discordsrv/listeners/PlayerAdvancementDoneListener.java

```diff
@@ -59,21 +64,47 @@ public void onPlayerAdvancementDone(PlayerAdvancementDoneEvent event) {
             return;
         }
 
+        String channelName = DiscordSRV.getPlugin().getMainTextChannel().getName();
```

Comment:
This would return the name of the channel on Discord, but you later retrieve the in-game channel using this variable.